### DISCUSSION TITLE
Theme for SSMS and Studio-3T.

### DISCUSSION THREAD

**Discussion by mddheeraj** [catppuccin/catppuccin#2069] [Port Requests]
Can we get themes for SSMS and studio 3T?

**Comment by sgoudham** [comment]
Hiya 👋 

It seems like you've been able to create a port request discussion without following the template (I assume you created it in another category and then switched it to this)

Please re-submit the request for theming `SSMS` and `Studio-3T` under 2 different discussions by following the template [here](https://github.com/catppuccin/catppuccin/discussions/new?category=port-requests). It helps us streamline the process and gives us the information that we need!
